subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,15 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files) found in inputs/outputs of tasks are handled gracefully from now on: similarly to missing files.
```

Comment:
```suggestion
Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files) found in inputs and outputs of tasks are handled gracefully from now on: as if they don't exist.
```

I don't think missing files is a term known to our users. If we want to make it so that they understand then we need to define this term in the user manual and link to it from here.